test/schema-feature.test.js

```diff
@@ -170,7 +170,7 @@ test('Cannot add schema for query and querystring', t => {
 
 test('Should throw of the schema does not exists in input', t => {
   t.plan(2)
-  const fastify = Fastify()
+  const fastify = Fastify({ exposeHeadRoutes: true })
```

Comment:
it is `exposeHeadRoutes` is `true` by default.

```suggestion
  const fastify = Fastify()
```

Reply:
Done in 0659eb5